### ISSUE TITLE
Unit tests for the Options API.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#21]
Originally reported on Google Code with ID 21

```
We have a bunch of unit tests for the Options API, but we need a lot more. In particular,
unit tests that deal with synchronization issues between frames.
```

Reported by `pcxunlimited` on 2010-09-06 01:16:29

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-06 01:16:44

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-06 04:36:37
- **Status changed**: `Delegated`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-12 02:08:40
- **Labels added**: Component-Testing

**Comment by Pauan** [comment]
```
Need unit tests so issue 16 doesn't happen again.

Also need unit tests for issue 34.
```

Reported by `pcxunlimited` on 2010-09-12 04:38:28
- **Blocked on**: #6, #4

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-19 18:34:48

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 21:56:53
- **Status changed**: `WontFix`
- **Blocked on**: #16, #34
- **No longer blocked on**: #16, #34
